eng/containers/ci.yml

```diff
@@ -28,6 +28,13 @@ parameters:
     dockerFile: 'tools/stress-cluster/services/Stress.Watcher/Dockerfile'
     stableTags:
     - 'latest'
+  - name: stress_common_tools
+    pool: 'ubuntu-20.04'
+    dockerRepo: 'engsys/eng-common-tools'
```

Comment:
```suggestion
    dockerRepo: 'stress/common-tools'
```

Reply:
@ckairen I would also move your deploy script into the stress-cluster directory alongside the Dockerfile and out of eng/common now that we're updating the approach.

Reply:
Dealt with here https://github.com/Azure/azure-sdk-tools/pull/2053